index.js

```diff
@@ -0,0 +1,239 @@
+const Twit = require("twit");
+const config = require("config");
+const winston = require("winston");
+require("winston-daily-rotate-file");
+const Client = require("bitcoin-core");
+const lbry = new Client({
+  version: "0.12.0",
```

Comment:
I don't think this is needed, is it?

Reply:
Correct, as i look through the library i can now see that it is not needed